### PR TITLE
Feature/userdata

### PR DESCRIPTION
**Pull request by TFuchsLa**

### CODE REVIEW

File: pages/api/plants/[id].js [hunk 1]
```diff
@@ -43,18 +43,27 @@ export default async function handler(req, res) {
...
+        return res
+          .status(403)
+          .json({ error: "Plants without an owner cannot be deleted" });
+      }
+
+      await Plant.findByIdAndDelete(id);
+
+      return res.status(200).json({ message: "Plant deleted successfully" });
     } catch (error) {
+      console.error("Error deleting plant:", error);
```
Comment (levinthiel) [inline pages/api/plants/[id].js:62]: You could delete this, right ?

File: pages/api/plants/index.js [hunk 2]
```diff
@@ -2,22 +2,31 @@ import dbConnect from "@/db/connect";
 import Plant from "@/db/models/Plant";
```
Comment (levinthiel) [inline pages/api/plants/index.js:2]: here I think you can remove the @

File: pages/api/reminders/index.js [hunk 3]
```diff
@@ -2,20 +2,31 @@ import dbConnect from "@/db/connect";
 import Reminder from "@/db/models/Reminder";
```
Comment (levinthiel) [inline pages/api/reminders/index.js:2]: here as well

**Review by levinthiel — APPROVED** [review]
Looks good guys, it's a green from me

File: pages/api/plants/[id].js (hunk 1 above)
Comment (ana-nova) [reply pages/api/plants/[id].js:62]: yes, thanks! we just forgot from the debugging

**Pull request merged**